src/index.js

```diff
@@ -4,24 +4,29 @@ const fetchStatuCakeData = require('./statuscake');
 const server = require('./server');
 const storage = require('./statuscakedata');
 
-const regex = /\s?([-])\s?/;
+const regexp = config.statuscake.regex;
 
-const getTagsFromTitle = Title => Title.split(regex);
+const getTagsFromTitle = title => title.match(new RegExp(regexp));
```

Comment:
Are we sure this function is doing want it's name means ?

Reply:
Exact 🍂